### PR TITLE
feat(schematics): add prompts

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

While running the schematics (`ng-add` or `generate`) we have to remember the options. 
If we don't add options, the schematic will do nothing.

Annotation 2019-12-24 131422

Annotation 2019-12-24 131619

Annotation 2019-12-24 131737

Issue Number: N/A

## What is the new behavior?

The user will be prompted if she doesn't provide the options:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Formatting is changed due to prettier.

### CODE REVIEW

**Comment by SanderElias** [comment]
Thank you @timdeschryver 
As @jorgeucano is the lead on schematics and is out on a holiday it may take a bit longer to merge this,

**Comment by timdeschryver** [comment]
Thanks for the update, there's no rush - enjoy the holidays 🎄

File: schematics/scully/src/create-markdown/schema.json [hunk 1]
```diff
@@ -12,11 +12,13 @@
     },
     "name": {
       "type": "string",
-      "description": "add the name for the folder and module"
+      "description": "add the name for the folder and module",
+      "x-prompt": "What should be the name for the folder and module?"
```
Comment (aaronfrost) [inline schematics/scully/src/create-markdown/schema.json]: grammar: `What name do you want to use for the folder and module?`

File: schematics/scully/src/add-post/schema.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-  }
+  "$schema": "http://json-schema.org/schema",
+  "id": "Scully-ng-add-blog",
+  "title": "Scully ng-add-blog schematic",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "add the title for the post",
+      "x-prompt": "What should be the title for the post?"
```
Comment (aaronfrost) [inline schematics/scully/src/add-post/schema.json]: grammar: `What should title do you want to use for the post?`

File: schematics/scully/src/create-markdown/schema.json [hunk 3]
```diff
@@ -12,11 +12,13 @@
...
     },
     "slug": {
       "type": "string",
-      "description": "add the name for the :${slug}"
+      "description": "add the name for the :${slug}",
+      "x-prompt": "What should be the slug for the markdown?"
```
Comment (aaronfrost) [inline schematics/scully/src/create-markdown/schema.json]: grammar: `What slug do you want for the markdown file?`

**Comment by aaronfrost** [comment]
@timdeschryver I made a few grammar suggestions.

**Comment by timdeschryver** [comment]
👍, thanks @aaronfrost !

File: schematics/scully/src/add-post/schema.json (hunk 2 above)
Comment (timdeschryver) [reply schematics/scully/src/add-post/schema.json]: I think there's a typo, it should be `What title do you want to use for the post` right?

**Pull request merged**